plugins/bundle/plugin.go

```diff
@@ -548,21 +549,24 @@ func (p *Plugin) saveBundleToDisk(name string, b *bundle.Bundle) error {
 	return os.Rename(tmpFile, bundleFile)
 }
 
-func saveCurrentBundleToDisk(path, filename string, b *bundle.Bundle) error {
-	var buf bytes.Buffer
-
-	if err := bundle.NewWriter(&buf).UseModulePath(true).Write(*b); err != nil {
-		return err
-	}
-
+func saveCurrentBundleToDisk(path, filename string, raw io.Reader) error {
 	if _, err := os.Stat(path); os.IsNotExist(err) {
 		err = os.MkdirAll(path, os.ModePerm)
 		if err != nil {
 			return err
 		}
 	}
 
-	return ioutil.WriteFile(filepath.Join(path, filename), buf.Bytes(), 0644)
+	if raw == nil {
+		return fmt.Errorf("no raw bundle bytes to persist to disk")
+	}
+
+	b, err := ioutil.ReadAll(raw)
+	if err != nil {
+		return err
+	}
+
+	return ioutil.WriteFile(filepath.Join(path, filename), b, 0644)
```

Comment:
[nit] this looks like we're bending ourselves to use ioutil.WriteFile. If instead we'd use `os.Open` on the file, we could use `io.Copy(dst io.Writer, src io.Reader)` to shovel the data into the file. What do you think?

Reply:
Done